python/dicebeard/__init__.py

```diff
@@ -121,10 +128,14 @@ def __init__(self, *args, **kwargs):
         # Report back to the user about their answer
         if answer == total:
             await self.sender.sendMessage(
-                '✅ Correct: {:.3}s'.format(timer.total_time))
+                'Correct: {:.3}s'.format(timer.total_time))
+            return timer.total_time
+            
         else:
             await self.sender.sendMessage(
-                '❌ Wrong: {:.3}s'.format(timer.total_time))
+                'Wrong: {:.3}s'.format(timer.total_time))
+            return 10     
```

Comment:
At the same time we also need to know whether the user was right or wrong. I find it unlikely that we will want to know the time spent on ones we get wrong?